README.md

```diff
@@ -12,3 +12,179 @@ the platform.
 **Breaking is bad**: Implementing *too many* new APIs before other browsers
 catch up, or failing to remove APIs other browsers don't intend to ship causes
 fragmentation.
+
+The purpose of *API Confluence Metrics* is to capture the ways in which
+different browsers risk breaking the elastic band.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [The Catalog](#the-catalog)
+- [The Metrics](#the-metrics)
+  - [API Velocity](#api-velocity)
+    - [Definition](#definition)
+    - [Rationale](#rationale)
+  - [Failure to Ship](#failure-to-ship)
+    - [Definition](#definition-1)
```

Comment:
Is there a way of giving these sections better tags? Links into this document will break if things are reordered.